app/config/PhaseConfig.scala

```diff
@@ -31,8 +37,24 @@ trait PhaseConfig {
   val maxPostcodeLength: Int
 }
 
-class TransitionConfig() extends PhaseConfig {
-  override val phase: Phase = Transition
+object PhaseConfig {
+
+  case class Values(apiVersion: Double)
```

Comment:
Appreciate it's not part of this ticket but it would be nice to add the existing config values to this case class - `maxNameLength`, `maxNumberAndStreetLength` and 'maxPostcodeLength' if you'd be able to do that?

Reply:
@karahanozturk spot on 👍